sdk/include/xdk/cmtypes.h

```diff
@@ -259,6 +259,9 @@ typedef enum _CM_SHARE_DISPOSITION {
 #define CM_RESOURCE_INTERRUPT_LATCHED         0x0001
 #define CM_RESOURCE_INTERRUPT_MESSAGE         0x0002
 #define CM_RESOURCE_INTERRUPT_POLICY_INCLUDED 0x0004
+#define CM_RESOURCE_INTERRUPT_ALLOW_RESERVED_IDT    0x0008
```

Comment:
The others are fine, this isn't in MS's wdm.h. Only in `miniport.h`.

Reply:
Shall I comment it out, with a comment? Or simply remove it?
In both files?

Reply:
@ThFabba : I think we would need something like `$ifndef (_WDMDDK_)` or similar, but that hpp command doesn't exist.

Reply:
There's no other XDK-generated header that it does go in. So an `ifdef` doesn't seem to make sense.
I'd say just remove it from here and keep it in NDK.

Reply:
Removed from XDK.